index.js

```diff
@@ -1,4 +1,5 @@
 /**
+ * @typedef {object} Declaration
```

Comment:
When I log the `node` value, the object type given is `Declaration`. Not sure where VS code is getting this type, but a person with better knowledge of importing available types into JSDocs might know.

<img width="681" alt="image" src="https://github.com/niksy/stylelint-no-restricted-syntax/assets/925735/2b188642-80cf-48b5-90d7-be36df965017">